packages/reports/addon/components/filter-values/dimension-select.js

```diff
@@ -16,6 +16,7 @@ import { inject as service } from '@ember/service';
 import Component from '@ember/component';
 import { get, set, computed, action } from '@ember/object';
 import { readOnly } from '@ember/object/computed';
+import { CARDINALITY_SIZES } from 'navi-data/models/metadata/dimension';
```

Comment:
`../`

Reply:
This one is in reports. I feel like it makes sense to leave it as is.

Reply:
Oops, missed that